aten/src/ATen/native/SummaryOps.cpp

```diff
@@ -52,10 +52,107 @@ Tensor _bincount_cpu_template(
   }
   return output;
 }
+
+// A helper function to get the bin for histogram
+// while each bin is inclusive at the lower end and exclusive at the higher,
+// i.e. [start, end) the last bin is inclusive at both, i.e. [start, end], in
+// order to include maxvalue if exists
+template <typename input_t>
+inline int64_t getbin(input_t x, input_t min, input_t max, int64_t nbins) {
+  if (x >= max)
```

Comment:
Braces for single line if statements like this, too, otherwise the internal linter will complain.